PYTHONPATH=~/tmp/miniconda3/envs/MaProject/bin/python:
python: can't open file '/home/anhcao/tmp/GitHub/DLProject_1/ModelTraining/train_continual.py': [Errno 2] No such file or directory
Launching Continual Learning Training Pipeline...
Traceback (most recent call last):
  File "/home/anhcao/tmp/GitHub/DLProject_1/ModelTraining/Wafer-map/main_train.py", line 36, in <module>
    main()
  File "/home/anhcao/tmp/GitHub/DLProject_1/ModelTraining/Wafer-map/main_train.py", line 30, in main
    subprocess.run(["python", "train_continual.py", "--config", args.config], check=True)
  File "/home/anhcao/tmp/miniconda3/envs/MaProject/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'train_continual.py', '--config', 'Wafer-map/config.yaml']' returned non-zero exit status 2.
